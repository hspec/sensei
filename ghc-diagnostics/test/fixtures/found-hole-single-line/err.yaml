version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: A
  start:
    line: 3
    column: 5
  end:
    line: 3
    column: 6
severity: Error
code: 88464
message:
- 'Found hole: _ :: A'
- 'In an equation for `a'': a = _'
- |-
  Relevant bindings include a :: A (bound at A:3:1)
  Valid hole fits include a :: A (bound at A:3:1)
hints: []
