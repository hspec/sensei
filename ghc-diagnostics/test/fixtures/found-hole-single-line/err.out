A:3:5: error: [GHC-88464]
    • Found hole: _ :: A
    • In an equation for `a': a = _
    • Relevant bindings include a :: A (bound at A:3:1)
      Valid hole fits include a :: A (bound at A:3:1)

